lib/commons/dom/is-offscreen.js

```diff
@@ -35,6 +35,10 @@ function isOffscreen(element, { isAncestor } = {}) {
   element =
     element instanceof AbstractVirtualNode ? element.actualNode : element;
 
+  if (!element) {
+    return undefined;
```

Comment:
Are  you sure changing the return type doesn't matter here? If so, update the docblock too.